descriptors/ModuleDescriptor-template.json

```diff
@@ -1102,10 +1114,18 @@
       "id": "finance.transactions",
       "version": "5.1"
     },
+    {
+      "id": "finance.fiscal-years",
+      "version": "2.0"
+    },
     {
       "id": "finance.ledgers",
       "version": "2.0"
     },
+    {
+      "id": "finance-storage.ledgers",
+      "version": "4.0"
+    },
     {
```

Comment:
finance-storage.order-transaction-summaries what about this permissoin, it is required or we can skip this

Reply:
after introducing new Batch Transaction API we don't use transaction-summaries endpoints from mod-orders any more, so it is not needed. For Q release these endpoints will be considered as deprecated, and for R we will completely remove them